src/internet_identity/tests/integration/v2_api/identity_register.rs

```diff
@@ -37,6 +38,40 @@ fn should_register_multiple_identities() {
     assert_ne!(identity_number_1, identity_number_2);
 }
 
+#[test]
+fn should_transition_flow_principal_to_temp_key() {
```

Comment:
Why this name for this test?

It's to make sure that the temp_key is expired after the required time, right?

Reply:
The test is there for two things:
1. the principal used in the flow can still be used after the registration (e.g. to add additional devices / get a delegation, etc.)
2. the principal used in the flow expires after the expected amount of time